dashboard/test/controllers/api/v1/users_controller_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class Api::V1::UsersControllerTest < ActionController::TestCase
+  setup do
+    @user = create :user
+  end
+
+  test 'a get request to using_text_mode returns using_text_mode attribute of user object' do
+    sign_in(@user)
+    get :get_using_text_mode, params: {user_id: 'me'}
+    assert_response :success
+    response = JSON.parse(@response.body)
+    assert !response["using_text_mode"]
```

Comment:
Do you want to accept nil values? Incidentally this will also succeed if the hash is missing that key.

To be more strict, use `assert_equal false, response["using_text_mode"]`